app/main.py

```diff
@@ -2,0 +1,37 @@
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
```

Comment:
Code Style issue: Use descriptive and correct variable names. The `name` attribute is a good example of a descriptive and correct variable name.